src/lib.rs

```diff
@@ -1113,6 +1123,17 @@ impl Display {
         ops::read_from_default_fb(gl::FRONT_LEFT, self)
     }
 
+    /// Execute an arbitrary closure with the OpenGL context active. Useful if another
+    /// component needs to directly manipulate OpenGL state.
```

Comment:
And please add a warning saying that the function must preserve the OpenGL state. Glium will just totally break if this isn't respected.